deblur.log

```diff
@@ -0,0 +1,11 @@
+INFO(140629163607872)2019-10-21 09:49:33,995:*************************
```

Comment:
A log file should probably not be in the PR. Is there any way you could move it into its own folder and then add that to the .gitignore?

Reply:
This log file has been the bane of my existence. It will get re-created there the next time deblur is run even if I move it.

Reply:
That is annoying, well it's a small issue and I trust you've tried the easy ways of remedying it.